core/src/main/java/nu/yona/server/crypto/pubkey/Encryptor.java

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2015, 2016 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License,
  * v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
  *******************************************************************************/
-package nu.yona.server.crypto;
+package nu.yona.server.crypto.pubkey;
 
 public interface Encryptor
```

Comment:
We should maybe use 'encrypter'/'decrypter' instead of 'encryptor'/'decryptor'. The -or ending is less common, see https://en.oxforddictionaries.com/spelling/nouns-ending-in-er-or-and-ar. Currently we are using both by the way (e.g. `StringFieldEncrypter`.